action.yml

```diff
@@ -25,10 +24,6 @@ inputs:
     description: |
       The name of the event that triggered the workflow
     default: ${{ github.event_name }}
-  use_nimma:
```

Comment:
https://github.com/stoplightio/spectral/commit/3af8b69e21ac73493c7539ddb0b8970782fcd5ac true by default, cannot change it.